src/renderer/dx/CustomTextLayout.cpp

```diff
@@ -508,9 +555,6 @@ try
 
     // We're going to walk through and check for advances that don't match the space that we expect to give out.
 
-    DWRITE_FONT_METRICS1 metrics;

```

Comment:
I noticed that no one is actually using this so I moved it out.

Reply:
Works for me, thanks.